src/contrib/cluster/Akka.DistributedData.LightningDB/LmdbDurableStore.cs

```diff
@@ -80,11 +82,16 @@ public LmdbDurableStore(Config config)
                     TimeSpan.Zero :
                     _config.GetTimeSpan("write-behind-interval");
 
+            _mapSize = _config.GetByteSize("map-size") ?? 100 * 1024 * 1024;
+
             var path = _config.GetString("dir");
             _dir = path.EndsWith(DatabaseName)
                 ? Path.GetFullPath($"{path}-{Context.System.Name}-{Self.Path.Parent.Name}-{Cluster.Cluster.Get(Context.System).SelfAddress.Port}")
                 : Path.GetFullPath(path);
 
+            _dirExists = Directory.Exists(_dir);
+
+            _log.Info($"Using durable data in LMDB directory [{_dir}]");
```

Comment:
Logging moved here so that it only logged once at startup, not every time data store is invoked.